src/analysis/retail/shaman/restoration/CHANGELOG.tsx

```diff
@@ -1,48 +1,14 @@
 import { change, date } from 'common/changelog';
-import SPELLS from 'common/SPELLS';
-import TALENTS from 'common/TALENTS/shaman';
+//import SPELLS from 'common/SPELLS';
+//import TALENTS from 'common/TALENTS/shaman';
 import {
-  Arlie,
-  Elitesparkle,
-  Fassbrause,
-  niseko, Seriousnes,
-  ToppleTheNun,
-  Vetyst,
-  Vohrr,
+  Seriousnes,
   Ypp,
 } from 'CONTRIBUTORS';
-import { SpellLink } from 'interface';
+//import { SpellLink } from 'interface';
```

Comment:
Please just remove unneeded imports, they can be added back in when they're needed